src/main.rs

```diff
@@ -342,3 +307,75 @@ fn err_to_warp(e: podsync::Error) -> impl warp::Reply {
 pub struct QuerySince {
     since: crate::time::Timestamp,
 }
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
```

Comment:
Do we need the `Eq` traits here?

Reply:
Not really; I often include them for future use, but since this is internal, you are completely correct that it is superfluous.